udata/core/storages/utils.py

```diff
@@ -41,3 +39,17 @@ def crc32(file):
 def mime(url):
     '''Get the mimetype from an url or a filename'''
     return mimetypes.guess_type(url)[0]
+
+
+def extension(filename):
+    '''Properly extract the extension from filename'''
```

Comment:
Cool, we had problems with that 👍

Reply:
I think we'll still have problems with that kind of filenames though `Capture d'écran 2018-02-13 16.03.07.png`. Maybe check against a list of known extensions and if found do not try to parse a compounded one? But it could get complicated.

Reply:
Yes, I didn't changed anything, I just grouped an existing function.
I think I can improve the behavior, but in another PR